app/liquid_tags/emgithub_tag.rb

```diff
@@ -0,0 +1,56 @@
+require "cgi"
+
+class EmgithubTag < LiquidTagBase
+  PARTIAL = "liquids/emgithub".freeze
+  OPTION_REGEXP = /\A(showBorder|showLineNumbers|showFileMeta)*\z/.freeze
+  LINK_REGEXP = %r{\A(http|https)://github.com/[a-zA-Z0-9]*/[a-zA-Z0-9]*/blob/[a-zA-Z0-9_./#-]*\z}.freeze
```

Comment:
Mmm this regexp would also match something like `https://github.com///blob/`

It's probably easier to just check if the URL starts with github.com, it's followed by `/\w+/\w+/blob/` and the optional chars you put at the end. Remember that `*` means 0 ore more, we need at least 1 per each part of the path. You can test on https://rubular.com/